### PR TITLE
[BALDE-39] Set up release build on travis

### PR DESCRIPTION
**Pull request by BIGbadEL**

### CODE REVIEW

File: src/utils/file_manager.cpp [hunk 1]
```diff
@@ -31,16 +31,17 @@ namespace Bald::Utils {
...
+                delete[] buffer;
+                fclose(file);
+                return result;
+            }
             delete[] buffer;
-
             fclose(file);
-
-            return result;
+            CORE_LOG_ERROR("[FILE_MANAGER] error while reading file");
```
Comment (Blinkuu) [inline src/utils/file_manager.cpp]: Change this to: CORE_LOG_ERROR("[FILE_MANAGER] Error while reading file");

**Review by Blinkuu — CHANGES_REQUESTED** [review]

File: src/utils/file_manager.cpp (hunk 1 above)
Comment (BIGbadEL) [reply src/utils/file_manager.cpp]: wrr

**Review by piotrmoszkowicz — CHANGES_REQUESTED** [review]
Why do you use float eq when you are comparing length of vectors? (ASSERT_FLOAT_EQ to ASSERT_EQ)

**Comment by BIGbadEL** [comment]
> Why do you use float eq when you are comparing length of vectors? (ASSERT_FLOAT_EQ to ASSERT_EQ)

because length is a float

**Review by Blinkuu — APPROVED** [review]
Good

File: src/utils/file_manager.cpp (hunk 1 above)
Comment (Blinkuu) [reply src/utils/file_manager.cpp]: <3

**Comment by piotrmoszkowicz** [comment]
Ah yeah, that's right, things happen when you make code review on phone 😅

**Review by piotrmoszkowicz — APPROVED** [review]

**Pull request merged**